automated_packaging/common_functions.pm

```diff
@@ -150,11 +150,12 @@ sub create_release_changelog {
             foreach $line (@log_output) {
                 if ($line =~ /^DESCRIPTION: */) {
                     $description_part = substr($line, length($&), -1);
+                    
                     if (length($description_part) > 78) {
-                        print("You have to shorten PR message $description_part of $pr_url");
-                        `git reset --hard`;
-                        die "Can not add description longer than 78 charachters";
+                        print("You have to shorten PR message $description_part of $pr_url\n");
+                        print("Description should not be longer than 78 charachters, please manually shorten this description\n");
```

Comment:
```suggestion
                        print("Description should not be longer than 78 charachters, please manually shorten this description\n");
                        push(@comment_lines, "* " . "PLEASE SHORTEN THE NEXT LINE MANUALLY, IT SHOULD BE NO LONGER THAN 78 CHARS\n");
```

Reply:
Oh now I understood what you purposed -adding those warnings to changelog as well-, makes sense